src/main/java/org/commcare/formplayer/application/FormController.java

```diff
@@ -66,29 +67,58 @@
 @EnableAutoConfiguration
 public class FormController extends AbstractBaseController {
 
+    private final Log log = LogFactory.getLog(FormController.class);
+    private final ObjectMapper mapper = new ObjectMapper();
+    @Autowired
+    private MediaMetaDataService mediaMetaDataService;
     @Autowired
     private SubmitService submitService;
-
     @Autowired
     private CategoryTimingHelper categoryTimingHelper;
-
     @Autowired
     private FormplayerStorageFactory storageFactory;
-
     @Autowired
     private RedisTemplate redisVolatilityDict;
-
     @Autowired
     private FormplayerDatadog datadog;
-
     @Resource(name = "redisVolatilityDict")
     private ValueOperations<String, FormVolatilityRecord> volatilityCache;
-
     @Value("${commcarehq.host}")
     private String host;
 
-    private final Log log = LogFactory.getLog(FormController.class);
-    private final ObjectMapper mapper = new ObjectMapper();
+    // Iterate over all answers and attempt to save them to check for validity.
+    public static HashMap<String, ErrorBean> validateAnswers(FormEntryController formEntryController,
```

Comment:
it looks like the whole class got reformatted. It's fine for now but generally the best practice on formatting is to only re-format the code that you change in the PR to facilititate easier review.

Reply:
That makes sense. I'll make sure to only reformat the changes I'm making going forward.